src/bindings/python/fluxacct/accounting/job_archive_interface.py

```diff
@@ -123,7 +123,13 @@ def add_job_records(rows):
     job_records = []
 
     for row in rows:
-        rset = ResourceSet(row[6])  # fetch R
+        try:
+            # attempt to create a ResourceSet from R
+            rset = ResourceSet(row[6])
+            nnodes = rset.nnodes
+        except (ValueError, TypeError):
+            # can't convert R to a ResourceSet object; skip it
+            continue
```

Comment:
Is skipping it a better option than using an empty `ResourceSet` or something? Not like I would know, just curious.

Reply:
I think in flux-accounting's case, it probably would be best to skip, the reason being that it needs information from `ResourceSet` (specifically `nnodes`) to calculate job usage, so if it can't retrieve that information from the job, it should not be factored in.

This sort of check is kind of reinforcing the check made in the script that actually fetches job records from Flux using the `job-list` module, which already filters job that have actually run. So while I don't expect this kind of `Exception` to happen very often, I think it would be best to just skip adding the job if it does occur.

Reply:
Sounds good!